templates/default/cassandra-env.sh.erb

```diff
@@ -188,6 +188,11 @@ then
 fi
 <% end -%>
 
+<% if node['cassandra']['priam']['version'] != '' -%>
+# add the priam cass extensions javaagent
+  JVM_OPTS="$JVM_OPTS -javaagent:$CASSANDRA_HOME/lib/priam-cass-extensions-<%= node['cassandra']['priam']['version'] -%>.jar"
```

Comment:
C* comes with a default javaagent `jamm`, if user want to use `priam` instead, `jamm` likely to be disabled. not sure C* supports multiple `javaagent`.